src/main/java/student/Student.java

```diff
@@ -0,0 +1,40 @@
+package student;
+
+public class Student {
+    private String name;
+    private int id;
+    private float gpa;

```

Comment:
Since gpa is going to be a precise decimal value that goes up to 2 decimal points, would it be better if we replace `float` to `BigDecimal`? This will reduce the potential bugs that may occur when we are doing our comparison of gpas in our algorithm.

Reply:
I saw this in the website under Week 2 on Java: Data Types and thought maybe it's a good point to bring up.

Reply:
That's a great point! We should indeed make use of `BigDecimal`

Reply:
https://www.geeksforgeeks.org/bigdecimal-class-java/

im not sure whether we should bring in a new package when floats or doubles would probably suffice for two decimal points which we will likely not do any arithmetic calculations to alter the values, likely only comparison is necessary which to and there are ways to compare accurately as well like

`public class FloatComparison {
    public static boolean areEqual(float num1, float num2, float epsilon) {
        return Math.abs(num1 - num2) < epsilon;
    }

    public static void main(String[] args) {
        float num1 = 1.2345f;
        float num2 = 1.2346f;

        // Use a small epsilon, e.g., 0.01f for two decimal places
        float epsilon = 0.01f;

        if (areEqual(num1, num2, epsilon)) {
            System.out.println("The numbers are equal up to two decimal places.");
        } else {
            System.out.println("The numbers are not equal.");
        }
    }
}
`

What does the team think? >.<

Reply:
Let's move this into the issues channel!